docs/ViewModel.md

```diff
@@ -7,22 +7,39 @@ Provides or describes a constructor function that provides values and methods
 to the component’s [can-component::view view]. The constructor function
 is initialized with values specified by the component element’s [can-stache-bindings data bindings].
 
-@option {function} A constructor function usually defined by [can-define/map/map.extend DefineMap.extend] or
+@option {function} A constructor function (usually defined by [can-define/map/map.extend DefineMap.extend] or
 [can-map Map.extend] that will be used to create a new observable instance accessible by
-the component’s [can-component::view].
+the component’s [can-component::view]) or an Object that will be passed to [can-define/map/map.extend DefineMap.extend].
 
 For example, every time `<my-tag>` is found, a new instance of `MyTagViewModel` will
 be created:
 
 ```js
+import Component from "can-component";
+import DefineMap from "can-define/map/map";
+
 const MyTagViewModel = DefineMap.extend( "MyTagViewModel", {
 	message: "string"
 } );
 
 Component.extend( {
 	tag: "my-tag",
 	ViewModel: MyTagViewModel,
-	view: stache( "<h1>{{message}}</h1>" )
+	view: "<h1>{{message}}</h1>"
+} );
+```
+
+The example above can be abbreviated to:
+
+```js
+import Component from "can-component";
+
+Component.extend( {
+	tag: "my-tag",
+	ViewModel: {
+		message: "string"
+	},
+	view: "<h1>{{message}}</h1>"
```

Comment:
I tried to make the signature show `Object|function` but the formatting was completely wrong when I did that.

Here’s what I ended up with:
<img width="817" alt="screen shot 2018-06-21 at 10 02 48 pm" src="https://user-images.githubusercontent.com/10070176/41758629-e0c02630-759e-11e8-819f-a45d60fccf1a.png">

Reply:
I think this is good. 👍.